engine/class_modules/priest/sc_priest.cpp

```diff
@@ -141,6 +141,11 @@ struct mind_blast_t final : public priest_spell_t
 
       priest().buffs.coalescing_shadows->expire();
     }
+
+    if ( priest().talents.discipline.harsh_discipline.enabled() )
+    {
+      priest().buffs.harsh_discipline->increment();
```

Comment:
I would use `trigger()` here instead, functionally they are the same in this case.